### PR TITLE
Updated existing slides

### PR DESCRIPTION
**Pull request by niksirbi**
Updated existing slides for this year's edition of the course.
Still missing some slides on keypoint-moseq theory and practice.
The notebooks will be updated in a separate PR.

## How to review
- Install `quarto` CLI and VSCode extension.
- Set the `QUARTO_PYTHON` env variable to point to a Python executable (in an env that contains `pandas`).
- Open the `index.qmd` file and render it inside VSCode (Cmd + Shift + K)

### CODE REVIEW

File: img/mouse-appearance-based-tracking.png
Comment (lochhh) [inline img/mouse-appearance-based-tracking.png:1]: The "apperance" typo is still not fixed

File: img/mouse-centroid-tracking.png
Comment (lochhh) [inline img/mouse-centroid-tracking.png:1]: multi-animal? (not important as it isn't used in the slides)

File: slides/SLEAP.qmd [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+::: aside
+see also the SLEAP [model evaluation notebook](https://sleap.ai/notebooks/Model_evaluation.html)
+:::
+
+## Run inference on new frames
+![](img/SLEAP_screenshots/9_inference.png){fig-align="center" height="400px"}
+
+::: aside
+- Expecte errors on silicon (M1/M2) Macs, see [this discussion](https://github.com/talmolab/sleap/discussions/1151)
```
Comment (lochhh) [inline slides/SLEAP.qmd]:
```suggestion
- Expect errors on silicon (M1/M2) Macs, see [this discussion](https://github.com/talmolab/sleap/discussions/1151)
```

File: slides/SLEAP.qmd [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+- Visualise and inspect data
+- Clean trajectories:
+  - Identify and drop outliers
+  - Smooth trajectories
+  - Interpolate over missing data
+- Compute variables of interest:
+  - Velocity, acceleration, heading, etc.
+  - Distances/angles between body parts
+  - Time spent in different regions
+  - Application-specific: navigation, social interactions, gait analysis
```
Comment (lochhh) [inline slides/SLEAP.qmd:196]: This part "overflows" on my screen. Could we make the list shorter or make this slide scrollable?

**Review by lochhh — APPROVED** [review]
🚀 Just a couple of typos

File: img/mouse-centroid-tracking.png
Comment (niksirbi) [reply img/mouse-centroid-tracking.png:1]: thanks for spotting, that image is no longer necessary and I've removed it.

File: img/mouse-appearance-based-tracking.png
Comment (niksirbi) [reply img/mouse-appearance-based-tracking.png:1]: fixed now.

File: slides/SLEAP.qmd (hunk 2 above)
Comment (niksirbi) [reply slides/SLEAP.qmd:196]: made this slide scrollable

**Pull request merged**